hack/verify-container-images.sh

```diff
@@ -51,5 +57,21 @@ rm ${TOOL_BIN}/trivy.tar.gz
 ## Builds the container images to be scanned
 make REGISTRY=gcr.io/k8s-staging-cluster-api-aws PULL_POLICY=IfNotPresent TAG=dev docker-build
 
+BRed='\033[1;31m'
+BGreen='\033[1;32m'
+NC='\033[0m' # No
+
 # Scan the images
-${TOOL_BIN}/trivy image -q gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller-${GO_ARCH}:dev
\ No newline at end of file
+echo -e "\n${BGreen}List of dependencies that can bumped to fix the vulnerabilities:${NC}"
+${TOOL_BIN}/trivy image -q --exit-code 1 --ignore-unfixed --severity MEDIUM,HIGH,CRITICAL gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller-${GO_ARCH}:dev && R1=$? || R1=$?
+echo -e "\n${BGreen}List of dependencies having fixes/no fixes for review only:${NC}"
+${TOOL_BIN}/trivy image -q  --severity MEDIUM,HIGH,CRITICAL gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller-${GO_ARCH}:dev
+
+if [ "$R1" -ne "0" ]
+then
+  echo -e "\n${BRed}Container images check failed! There are vulnerability to be fixed${NC}"
+  exit 1
+fi
+
+echo -e "\n${BGreen}Container images check failed! No unfixed vulnerability found${NC}"
```

Comment:
Should this be the `Container images check passed. No unfixed vulnerabilities found.`

Reply:
Yeah sounds right 😅

Reply:
My fault  😅 . Fixed! https://github.com/kubernetes-sigs/cluster-api-provider-aws/compare/c2e1cbaf7dcc081a136c438dc18df8e1c8acf660..e34e1b860da253c3cb077979bf4a022db014ef5f.